pkg/icingadb/cleanup.go

```diff
@@ -41,19 +44,47 @@ func (db *DB) CleanupOlderThan(
 	count uint64, olderThan time.Time, onSuccess ...OnSuccess[struct{}],
 ) (uint64, error) {
 	var counter com.Counter
-	defer db.log(ctx, stmt.Build(db.DriverName(), 0), &counter).Stop()
+	var n int64
```

Comment:
Might be pedantic, but `n` can be moved down in the `for` loop's scope and maybe renamed to something more obvious at first glance, e.g., `currentRows`.